pkg/didmethod/peer/did.go

```diff
@@ -25,8 +25,23 @@ const (
 	peerPrefix = "did:peer:"
 )
 
-//newDid creates the peer DID. For example : did:peer:11-479cbc07c3f991725836a3aa2a581ca2029198aa420b9d99bc0e131d9f3e2cbe
-func newDid(doc *did.Doc) (string, error) {
+//NewDID returns the doc with the computed peer did embedded in the genesis document
```

Comment:
To follow the spec's words: NewDID returns the resolved variant of the genesis version of the peer DID document